### ISSUE TITLE
Remove pipes from world map

### ISSUE THREAD

**Issue by Quintus** [Secretchronicles/TSC#423]
Even with ticket #406 closed soon, we ought to remove the pipes from the world maps entirely. It is not our style anymore, and while they may be used on special occasions, they should not be used to plaster the entire map with them, which just looks like Mario.

For this to work out properly, we’re going to need a lot more worldmap decoration than we have now. The state of the world map assets in TSC is just poor at the moment.

Valete,
Quintus

**Comment by Luiji** [comment]
I agree. I was about to comment on #406 that the amount of pipes currently on-screen doesn't really match the levels. I don't think it even matched SMC, since the style's always been a bit more natural looking (in my opinion).

**Comment by datahead8888** [comment]
Eventually there won't be many pipes in levels at all.  We will only have them in levels such as factories were they seem natural.  Otherwise, we'd have things like logs, teleporters, etc.  Thus this will make sense for the map.  We may consider waiting until the levels have had their pipes removed, however.

**Comment by datahead8888** [comment]
Also...

> The state of the world map assets in TSC is just poor at the moment.

At this link @Bugsbane suggests that the quality of most existing map graphics is poor:
http://forum.secretchronicles.de/topics/53

Bugsbane said:
> I think this is someone not liking the look of some of the art, but not having the skills to figure out exactly why. Personally, I think that most of the current overworld graphics are terrible. The problem isn't their cartoon style though. The problem is that they're not skillfully drawn. If redone, we could make the overworld look *drastically* better.

I think it's also worth evaluating what style we want on the world map before any major revision effort is done.  We don't have to continue looking like SMC, either.  This probably won't go anywhere until we have more artists on the team again, though, and it would deserve its own task.

**Comment by Quintus** [comment]
No artist available; postponing.
